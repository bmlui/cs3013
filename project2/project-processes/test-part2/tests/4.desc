Test use of absolute path to binary
