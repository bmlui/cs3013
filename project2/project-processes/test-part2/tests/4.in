/bin/ls
/bin/cat tests/4a.in
exit

