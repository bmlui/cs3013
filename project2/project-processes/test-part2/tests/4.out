test-part2$ /bin/ls
test_part2.sh
tests
tests-out
wshell
test-part2$ /bin/cat tests/4a.in
While safety is one of many Enrichment Center goals, the Aperture Science High
Energy Pellet, seen to the left of the chamber, can and has caused permanent
disabilities such as vaporization.
test-part2$ exit
